### DISCUSSION TITLE
VoidCallbackIntent: undefined_method error since last commit

### DISCUSSION THREAD

**Discussion by robkri** [bdlukaa/fluent_ui#541] [Q&A] [answered]
Since [this commit](https://github.com/bdlukaa/fluent_ui/commit/f61c8232d87e33e3b97236da9bd16ceb88a18b09), I get an "undefined_method" error in tree_view.dart (on lines 690 through 710) for VoidCallbackIntent which is - to my understanding - a part of [Flutter's widget package](https://api.flutter.dev/flutter/widgets/VoidCallbackIntent-class.html).

Now I don't know where the error is coming from: I'm using VS Code on Win10, and there where similar issues before, where a class suddenly didn't get recognized anymore. Usually restarting VS Code or calling `flutter pub get` fixed these things, but now it doesn't seem to help.

To make sure, this isn't an issue with my fork, I cloned the original repo again. Here I have the same problem.

Any ideas?

**Comment by robkri** [comment, marked as answer]
It was my mistake: My flutter version wasn't up to date...

After upgrading to 3.3 the problem was solved.

**Reply by umutocak** [reply]
I had the same problem. I'll update and test it tomorrow. Thank you.
